src/DynamoCoreWpf/Views/Search/LibraryView.xaml.cs

```diff
@@ -170,7 +171,10 @@ private void ExpandCategory(TreeViewItem sender, NodeCategoryViewModel selectedC
             // If categoryToBeExpanded is null, that means not category button, but class button was clicked.
             // During loop we will find out to which category this clicked class belongs.
             if (categoryToBeExpanded != null)
+            {
                 categoryToBeExpanded.IsExpanded = !categoryToBeExpanded.IsExpanded;
+                hasSelectedClassBeenFound = true;
+            }
```

Comment:
It can be false as well. Category can be clicked to open and to close. 
![image](https://cloud.githubusercontent.com/assets/8158404/6668539/471079e4-cbfc-11e4-8701-8cbf9fd5cf41.png)
In case above IsExpanded will be false.